app/src/main/java/com/android/streetworkapp/ui/progress/Progression.kt

```diff
@@ -0,0 +1,292 @@
+            tags = listOf("Exploration", "Beginner"),
+            description = "Granted for completing your first adventure in the wild."),
+        Achievement(
+            icon = R.drawable.park_default, // Replace with actual resource ID
+            title =
+                "First Place First Place First Place First Place First Place First Place First Place First Place",
+            tags =
+                listOf(
+                    "Victory",
+                    "Competition",
+                    "Gold",
+                    "Gold",
+                    "Victory",
+                    "Competition",
+                    "Gold",
+                    "Gold",
+                    "Victory",
+                    "Competition",
+                    "Gold",
+                    "Gold"),
+            description =
+                "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."),
+        Achievement(
+            icon = R.drawable.park_default, // Replace with actual resource ID
+            title = "Novice Explorer",
+            tags = listOf("Exploration", "Beginner"),
+            description = "Granted for completing your first adventure in the wild."),
+        Achievement(
+            icon = R.drawable.park_default, // Replace with actual resource ID
+            title = "Master of the Craft",
+            tags = listOf("Crafting", "Expert"),
+            description = "Recognized for mastering a complex crafting skill."),
+        Achievement(
+            icon = R.drawable.park_default, // Replace with actual resource ID
+            title = "Team Player",
+            tags = listOf("Collaboration", "Teamwork"),
+            description = "Earned by contributing significantly to a team project."))
+
+// note: I haven' tested big values in the metrics tabs, the assumption was that those shouldn't
+// overflow the boxes either way (unless we put some stupid score system)
+// I did test for the rest thought
+
+@OptIn(ExperimentalLayoutApi::class)
+@Composable
+fun ProgressScreen(
+    navigationActions: NavigationActions,
+    paddingValues: PaddingValues = PaddingValues(0.dp)
+) {
+  val progressBarSize = 145.dp
+
+  val scoreText = buildAnnotatedString {
+    append("780")
+
+    withStyle(style = SpanStyle(fontWeight = FontWeight.SemiBold)) { append("/1000") }
+  }
+
+  Box(modifier = Modifier.padding(paddingValues).testTag("progressionScreen")) {
+    LazyColumn(
+        modifier =
+            Modifier.fillMaxSize()
+                .padding(PaddingValues(0.dp, progressBarSize * 0.15f, 0.dp, 0.dp)),
```

Comment:
Suggestion: this is not a huge performance win but every recompositions this value is recomputed
You could mitigate this problem like this:
```kotlin
val columnPadding = PaddingValues(0.dp, Dimens.ProgressBarSize * 0.15f, 0.dp, 0.dp)
LazyColumn(
    modifier =
        Modifier.fillMaxSize().padding(columnPadding),
)
```

Reply:
done